roles/gateway/tasks/main.yml

```diff
@@ -289,3 +289,13 @@
   ansible.builtin.file:
     path: "{{ workingdir.path }}"
     state: absent
+
+- name: Uninstall gcc packages
+  ansible.builtin.dnf:
+    name: "{{ item }}"
+    allowerasing: true
+    autoremove: true
+    state: absent
+  with_items:
+    - gcc
+    - gcc-c++
```

Comment:
Should we create a new var called `gateway_build_packages` similar to the redis role and uninstall those packages instead?